src/System.Management.Automation/namespaces/FileSystemProvider.cs

```diff
@@ -2361,7 +2361,7 @@ public static string LastWriteTimeString(PSObject instance)
                     // non-existing targets on either Windows or Linux.
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
New-Item with links.New-Item -ItemType SymbolicLink should understand directory path ending with slash
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\New-Item.Tests.ps1: line 255
255:         $symbolicLinkPath | Should -Not -BeNullOrEmpty
```